### DISCUSSION TITLE
Export missing files

### DISCUSSION THREAD

**Discussion by mkoechli** [standardnotes/forum#2933] [General]
Hi there
I tested the new Super - editor (in my opinion, not yet up to his name ;-)). It is great having a way to integrate screenshots into notes. But now, if you try to export such a note (Markdown or HTML) the screenshots are lost.

Text in standardnotes
![grafik](https://user-images.githubusercontent.com/122264316/211273114-408c28b2-7383-4d2a-9141-a4244143630c.png)

Test in Markdown editor
![grafik](https://user-images.githubusercontent.com/122264316/211273213-ac8c93fd-0c14-4a64-b0b6-b364bc6b07e4.png)

[Sonntag, 8_ Jan_ 2023 at 16_25.md](https://github.com/standardnotes/forum/files/10371792/Sonntag.8_.Jan_.2023.at.16_25.md)

**Comment by effieeee** [comment]
Hi @mkoechli, this is the expected result at the moment since the images are only displayed inline through the direct link you see in the exported markdown file. As of now I'm afraid there aren't yet any concrete developments on including the decrypted images in the exported files.
